aio/content/guide/observables.md

```diff
@@ -1,6 +1,18 @@
 # Observables
 
-Observables support passing messages between publishers and subscribers in your application. This provides a nice decoupling between the producer and consumer of the messages.
+Observables provide support for passing messages between publishers and subscribers in your application. Observables offer significant benefits over other techniques for event handling, asynchronous programming, and handling multiple values. 
+
+Observables are declarative -- that is, you define a function for publishing values, but it is not executed until a consumer subscribes to it. The subscribed consumer then receives notifications until the function completes, or they unsubscribe. 
+
+An Observable can deliver multiple values of any type -- literals, messages, or events, depending on the context. The API for receiving values is the same whether the values are delivered synchronously or asynchronously. Because setup and teardown logic are both handled by the Observable, your application code only needs to worry about subscribing to consume values, and when done, unsubscribing. Whether the stream was keystrokes, an HTTP response, or an interval timer, the interface for listening to values and stopping listening is the same.
+
+Because of these advantages, Observables are used extensively within Angular, and are recommended for app development as well.
+
+## Basic terms 
```

Comment:
I'm not sure if this should be before or after the sample. The nice thing about this sample is that almost everything is defined in the code.

Reply:
Before. First you say what your example is meant to show, then you show it.
The point of documentation is to describe things in words, so that you know what to look for in the code samples.